Zend/zend_builtin_functions.c

```diff
@@ -728,8 +728,8 @@ ZEND_FUNCTION(get_class)
 		if (scope) {
 			RETURN_STR_COPY(scope->name);
 		} else {
-			zend_error(E_WARNING, "get_class() called without object from outside a class");
-			RETURN_FALSE;
+			zend_throw_error(NULL, "get_class() without arguments must be called from within a class");
```

Comment:
Was going to suggest that we drop support for this entirely, then saw that it's already on the roadmap: https://wiki.php.net/rfc/deprecations_php_8_0#get_class_without_argument